app.cpp

```diff
@@ -10,7 +11,8 @@ void main_task(intptr_t unused)
 
 void arm_task(intptr_t unused)
 {
-  // ext_tsk();
+  HorizonArm::keepArm();
```

Comment:
上手くできるかわからないけど、ここでループする方が、keepArm関数の再利用性が高くなると思う。
```suggestion
  while(true) {
    HorizonArm::keepArm();
  }
```